pages/common/elvish.md

```diff
@@ -0,0 +1,20 @@
+# elvish
+
+> Expressive programming language and a versatile interactive shell.
+> More information: <https://elv.sh/learn/>.
+
+- Start an interactive shell session:
+
+`elvish`
+
+- Execute a [c]ommand and then exit:
+
+`elvish -c "{{command}}"`
+
+- Execute a script:
```

Comment:
And exit?
Does it print what it is executing?

Reply:
Elvish Is a shell, I think the script decides that.

Reply:
No, elvish doesn't print executing script as cmd.exe does.